docs/using/syntax.md

```diff
@@ -721,9 +730,17 @@ $$ (eqn:best)
 
 This is the best equation {eq}`eqn:best`
 
+There are a few other options available to control dollar math parsing:
+
+`myst_dmath_allow_space=False`, will cause inline math to only be parsed if there are no initial / final spaces, e.g. `$a$` but not `$ a$` or `$a $`.
+
+`myst_dmath_allow_digits=False`, will cause inline math to only be parsed if there are no initial / final digits, e.g. `$a$` but not `1$a$` or `$a$2`.
```

Comment:
@chrisjsewell is this `on by default`?

Reply:
See the default configuration here: https://myst-parser.readthedocs.io/en/latest/using/intro.html#myst-configuration-options and in the code: https://github.com/executablebooks/MyST-Parser/blob/8206f2a6eb891737b0b7916b761d6a5311a08de4/myst_parser/main.py#L22

I'm open to discussion about these though 😄

Reply:
as in at present the default is that `$ a $` will be parsed as math

Reply:
ah nice page. I just noticed you added `(off by default)` in a couple other places in the text and was wondering if you wanted that here as well?

Reply:
Yeh good point

Reply:
If having a default value of `myst_dmath_allow_space=True` provides a more **robust** (but less technically correct) parse of the text -- I think it is probably good to have on by default. What do you think?

Reply:
> provides a more robust (but less technically correct) 

Well I don't know if there is a "technichally correct" syntax rule for this, since obviously its not covered by the CommonMark spec. Just to clarify, you think that `$ a $` should or should not be recognised a math?